great-idea/css/index.css

```diff
@@ -49,18 +49,18 @@ table {
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
 }
 
 html, body {
-    height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+	height: 100%;
+	font-family: 'Titillium Web', sans-serif;
 }
 
 h1, h2, h3, h4, h5 {
-    font-family: 'Bangers', cursive;
-    letter-spacing: 1px;
-    margin-bottom: 15px;
```

Comment:
I will go over it on monday(remind me if i forget haha) the TRBL for box model that will help with the margin and padding so you wont have to type left right top or bottom any more.